### ISSUE TITLE
[bug] Unable to set Associated Scene ID option in Scene Controller Configuration via Control Panel

### ISSUE THREAD

**Issue by pletch** [zwave-js/zwavejs2mqtt#1621]
### Version

**Checklist:**

- [x] I am **not** using HomeAssistant. **Or:** a developer has told me to come here.
- [x] I have checked the troubleshooting section and my problem is **not** described there.
- [x] I have read the changelog problem was **not** mentioned there.

Build/Run method

- [ ] Docker
- [ ] PKG
- [ ] Snap package
- [x] Manually built (git clone - yarn install - yarn run build )

zwavejs2mqtt: 5.5.1.a937e5e
zwave-js: 8.2.1

### Describe the bug
Attempts to Set the SceneID on a Leviton VRCS4 scene controller appear to work through the interface but the stored value on the device is never actually updated.  The driver returns the error: 
`Writing 6 to 33-45-0-sceneId-1 2021-08-27T18:09:40.079Z DRIVER Unexpected error during SerialAPI command: ZWaveError: Set commands don't support unknown durationsat Duration.serializeSet`

Along a similar vein, attempts to set dimmingDuration also fail on this same page with error in debug interface:
`Driver: Scene Controller Configuration: "dimmingDuration" is not a supported property`

### To Reproduce

1. Go to Control Panel.
2. Expand VRCS4 node details.
3. Scroll down and click to expand options for Scene Controller Configuration
4. Enter value in Associated Scene ID and press Arrow to send command.
5. Successfully call api writeValue is briefly displayed as notification.
6. Refresh page and observe that the Scene ID value is not actually updated.

### Expected behavior

sceneID value is updated to match sane numeric value entered in Scene Controller Configuration Associated Scene ID field.

### Additional context

[zwavejs_2021-08-27.log](https://github.com/zwave-js/zwavejs2mqtt/files/7068809/zwavejs_2021-08-27.log)
[zwavejs2mqtt.log](https://github.com/zwave-js/zwavejs2mqtt/files/7068811/zwavejs2mqtt.log)

**Comment by robertsLando** [comment]
I think you have to set a dimming duration in the input before the node values expandable panes, when that input is set the commands sent also have that property set

**Comment by pletch** [comment]
Thank you for the guidance but I am not sure I follow how this is supposed to work.  

This is the current configuration with no settings modified.
![scene configuration no options annotated](https://user-images.githubusercontent.com/17855860/131506976-f729a745-22ad-4c2d-9695-59205c493f36.JPG)

Scenario 1:  Changed first scene number from 6 to 5 and selected 1 second for dimmingDuration.  Pressed button next to scene number option. The Zwave2jsmqtt interface shows no error in this situation but the scene number is not actually updated from 6 to 5.
![scene configuration scenario 1 annotated](https://user-images.githubusercontent.com/17855860/131507246-fc7bed2a-8c2a-4f03-bb4a-1a486bea9bed.JPG)

Scenario 2:  Changed first scene ID from 6 to 5 and selected 1 second for dimmingDuration.  Selected button next to dimmingDuration option. In this case, the Zwavejs2mqtt interface shows error at top and the scene dimmingDuration is not updated.
![scene configuration scenario 2 annotated](https://user-images.githubusercontent.com/17855860/131507403-179197b2-7e07-447d-b06f-729fbbc0b126.JPG)

Attached logs show the sequence where Scenario 1 and Scenario 2 were tried.
[zwavejs2mqtt-scene.log](https://github.com/zwave-js/zwavejs2mqtt/files/7084121/zwavejs2mqtt-scene.log)
[zwavejs_2021-08-31.log](https://github.com/zwave-js/zwavejs2mqtt/files/7084122/zwavejs_2021-08-31.log)

**Comment by robertsLando** [comment]
I never used a scene controller so dunno how it is supposed to work. @blhoward2 @AlCalzone ?

**Comment by blhoward2** [comment]
I have two of these in a box somewhere and they don't work well that way, and didn't under SmartThings either. They can send out scene events you react to (if configured properly), or you can directly associate devices and they send out scene events to those devices. Setting up the scene IDs has only ever worked for me in Simplicity Studio. As I noted on the issue below a while back, you're not even seeing enough boxes as the device has 8 groups (one each for on and one for off on each button).

The only way to clear the lights, too, is with specific commands we don't support. https://github.com/zwave-js/node-zwave-js/issues/2397

These are first generation scene controllers and don't use central scene. They're terrible to configure in general and limited in what they can do. Frankly, you're better to swap them for something more modern like the Zooz ZEN32.

**Comment by pletch** [comment]
I don't interpret this to be commanding a scene.  This configuration option should be assigning a Scene ID number and/or dimmingDuration to the associated button on the controller per the Set method of the Scene Controller Configuration class
[https://zwave-js.github.io/node-zwave-js/#/api/CCs/SceneControllerConfiguration?id=scene-controller-configuration-cc-methods](url) but it doesn't seem to work.  

Doesn't the commandclass code [here](https://github.com/zwave-js/node-zwave-js/blob/master/packages/zwave-js/src/lib/commandclass/SceneControllerConfigurationCC.ts#L139) imply that a default dimmingDuration should be set if none is specified?  

@blhoward2   The device is indeed old but I like the form factor with distinct very visible button light colors and I remember how much I paid for it when I bought it nearly 10 years ago 😃 .  I am successfully catching the scene activated events in Home Assistant and have the button lights working through a separate script that sends the raw commands necessary to set the individual button light states.  I will take a look at the Zen32 and decide whether to abandon.

In actuality, this issue not all that significant for me really. I discovered when starting to investigate some of the scene configuration options described in [https://github.com/zwave-js/node-zwave-js/issues/2397](https://github.com/zwave-js/node-zwave-js/issues/2397).  

I only raised the issue as I suspected i'd stumbled on to some broken functionality and it should be fixed.

**Comment by blhoward2** [comment]
I edited my post before you posted, see above. We don’t even detect the right number of groups.

**Comment by blhoward2** [comment]
The ZEN32 is much easier to use and configure. It has a relay you can disable if you don't need. And the button LED colors and brightness are easily configurable via parameters, so no custom scripts needed. (I just use the HA service to set the LED to always on of always off depending on whether I want them on or off.)

The buttons are a tad daintier than the VRCS4, as those are built like a tank, but in my opinion the functionality makes it worth it.

**Comment by pletch** [comment]
Awesome, I ordered a Zen32.  

Edit: Removed question not at all relevant to the original issue details.

**Comment by blhoward2** [comment]
Yes, you'd just set it to always off, delay, then always on, etc. Or do the same with different colors. Depending on how you do this in an automation you can also set the mode to Single and it won't restart if it's in process. https://www.home-assistant.io/docs/automation/modes/

**Comment by AlCalzone** [comment]
Scene controller configuration is meant to configure the `Scene Activation CC` commands that are sent to the associated devices - not only to the controller.

Let's leave this open. Either the dimmingDuration should be actually writable (but the value API isn't meant to set multiple related values together) or it shouldn't (bug in node-zwave-js) and UIs should provide a proper way to configure the scenes on a remote device (issue for zwavejs2mqtt)

**Comment by Ikcelaks** [comment]
Setting the `dimmingDuration` should absolutely be part of the API.

The problem is that, when I originally added the `Scene Controller Configuration CC`, I did not know how to accept a `duration` value in the value API. I should be able to fix this easily.
